cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -261,6 +261,7 @@ func KnownControllers() []string {
 
 var ControllersDisabledByDefault = sets.NewString()
 
+// newControllerInitializers  will be all controller for unified management, unified initialization at start time
```

Comment:
I really appreciate the fact that you're trying to improve the documentation of our code. Unfortunately, this comment is not easy to understand. Ordinarily in this situation, I would offer a suggestion for improvement, but at the moment I'm just totally swamped. Thanks for understanding :(